style.css

```diff
@@ -17,7 +17,6 @@ body {
 .frame-container {
   display: flex;
   justify-content: space-between;
-
 }
 
 .description {
```

Comment:
- [ ] As requested by the previous, kindly make sure to stick to the Figma design by styling your page to look exactly the as the provided design. Your navigation bar is not aligned well and please make sure to use the correct fonts, font-size and make sure to provide the right spacing between the element and containers. :wink:
- [ ]  Kindly, make sure the background is placed on the page correctly as I the provided design :wink: 

  - ### Your design
  
  ![image](https://user-images.githubusercontent.com/87197412/185521565-caa9c7e7-98f0-4f8f-a237-b60ddc0447a3.png)

    - ### Figma design
   
  ![image](https://user-images.githubusercontent.com/87197412/185521657-4d8188c6-8780-4632-9344-a747eef20ee6.png)

Reply:
Please note that this is a mobile view. You have to reduce the screen size to see it. Ill address the welcome and hamburger alignment. But you need to compress the image to get the best view